evalml/tests/data_checks_tests/test_id_columns_data_check.py

```diff
@@ -125,21 +126,17 @@ def test_id_cols_data_check_input_formats():
     # test empty pd.DataFrame
     assert id_cols_check.validate(pd.DataFrame()) == {"warnings": [], "errors": []}
 
-    #  test list
-    assert id_cols_check.validate([1, 2, 3, 4, 5]) == {
```

Comment:
I think it's fine to remove this test case in favor of passing in a 2d dataframe or data table. But I wonder why we even supported it in the beginning? Was the goal to also check if a single column (or maybe the target values) were id?

Reply:
Yep I think this was just checking that a 1d vector would get treated just like a 2d dataframe with one column, right? I agree its fine to use 2d input here instead, although I do think it would be nice to have a test case with 1d input too.

Reply:
Yeah, I think so. That worked "fine" because we converted any input we got into a DataFrame. Current updated logic will see that this is 1D and update to a DataColumn, which will then convert back to a Series. Issue is, Series objects don't have some methods available, breaking our logic so I'm removing this. If we see it fit / useful to provide 1D inputs, we can add back? I didn't think it was worth cluttering the logic with extra dimension checks/different logic 🤔